setup.py

```diff
@@ -33,7 +33,8 @@ def get_snakemake_pkg(extras=""):
     "docs": ["Sphinx>=1.4.4", "sphinx-rtd-theme>=0.1.9",],
     "tests": tests_require,
     "kubernetes": ["kubernetes>=11.0.0,<12.0.0",],
-    "yadage": ["adage==0.10.1", "yadage==0.20.1", "yadage-schemas==0.10.6",],
+    # c.f. https://github.com/reanahub/reana-commons/issues/322
```

Comment:
I don't think we need this comment.

Reply:
Indeed, we don't need the reason here in the code, the change should be clear enough and we have the reason in the commit log and in the GitHub issue discussion anyway!

Two small things:

- Please add yourself to the `AUTHORS.rst`:

```
- `Matthew Feickert <https://orcid.org/0000-0003-4124-7862>`_
```

- Please add `Closes #322` to the commit log message.

Can you please amend those three things and squash into one commit and repush with force?

Thanks for the PR!

Reply:
Done! :+1: